docker-compose.yml

```diff
@@ -58,7 +58,7 @@ services:
       - RELEASE_BUMPER=foobar
 
   diagnostics:
-    image: nebraltd/hm-diag:d444f93
+    image: nebraltd/hm-diag:f476963
```

Comment:
This should point to https://github.com/NebraLtd/hm-diag/pull/211 after its merged.